simpleui/ViewController.swift

```diff
@@ -7,8 +7,112 @@
 //
 
 import UIKit
+import Darwin
+
 
 class ViewController: UIViewController {
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.view.backgroundColor = UIColor(red: 254 / 255.0, green: 249 / 255.0, blue: 55 / 255.0, alpha: 1 / 1)
+    }
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    var t_stat:Int = 4
```

Comment:
In Swift ist camelCase die Konvention, schreib also lieber z.B. `timeStat` oder dergleichen. Benne die Variablen außerdem etwas deskriptiver, sodass man direkt versteht, was sie repräsentieren.

Diese `Int` Variablen brauchst du auch gar nicht, oder? Es reichen wohl die `Double`s, sonst sind alle ja dupliziert.